packages/core/src/components/Onboarding/index.tsx

```diff
@@ -36,6 +37,14 @@ export function Onboarding({
 
   const [page, setPage] = useState<number>(platform === 'web' ? 1 : 0)
 
+  let names = namer(primaryColor)
```

Comment:
Do we need to define this variable here? I guess if its used only inside the useEffect, we can just define and use it there?
Can you please check once if it works this way?

Reply:
Agreed — i dont think this needs to be defined here.